app/style/main.less

```diff
@@ -94,9 +70,76 @@
 // Main scroll bar styling
 ::-webkit-scrollbar {
   width: 5px;
-  background-color: #e7eaed;
+  background-color: var(--disabled-background-color);
 }
 
 ::-webkit-scrollbar-thumb {
-  background-color: #8797a5;
+  background-color: var(--disabled-color);
+}
+
+.theme-dark {
+  --background-back-color: #4c4c4c; // all gone
+  --background-container: #616262; // all gone
+  --background-container-hover: #484949;
+  --background-hovered: #e9f8fe;
+  --input-color-default: #b5b8c5; // all gone
+  --input-color-hovered: #b5b8c5;
+  --input-color-focused: #b5b8c5;
+  --input-color-disabled: #dadfe2;
+  --stroke-color-default: #a9b4bf;  // all gone
+  --title-text-and-button-background: #b5bbc5;
+  --title-text-and-button-background-hovered: #b5bbc5;
+  --stroke-color-hovered: #bcc4cc;
+  --stroke-color-focused: #a9c5ff;
+  --input-color: #a9c5ff;
+  --input-color-hover: #7ea9ff;
+  --link-color: #a9c5ff;
+  --link-hover-color: #6397ff;
+  --accent-color: #2ed8a3;
+  --blue-highlight-background: #546065;
+  --tab-text: #9ca8b5;
+  --tab-text-active-hover: #c4ccd4;
+  --cyan-bold: #69d5f7;
+  --disabled-color: #8997a5;
+  --disabled-background-color: #e7eaed;
+  --error-text-color: #fd714b;
+  --error-red: red;
+  --vote-yes-color: #2ed8a3;
+  --vote-no-color: #f2734f;
+  --vote-abstain-color: #596d81;
+  --white-border: #d1d5db;
+}
+
+.theme-light {
+  --background-back-color: #fff; // all gone
```

Comment:
What do these "all gone" mean here?

Reply:
Ah apologies on these, some left over comments that I was using to be organized